openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -630,24 +627,26 @@ export const renderDataInsightLineChart = (
         type="category"
       />
       <YAxis
-        dataKey="count"
-        tickFormatter={(value: number) => axisTickFormatter(value, '%')}
+        tickFormatter={
+          isPercentage
+            ? (value: number) => axisTickFormatter(value, '%')
+            : undefined
+        }
       />
 
-      {graphData.map((s, i) => (
+      {labels.map((s, i) => (
         <Line
-          data={s.data}
-          dataKey="count"
+          dataKey={s}
           hide={
-            activeKeys.length && s.name !== activeMouseHoverKey
-              ? !activeKeys.includes(s.name)
+            activeKeys.length && s !== activeMouseHoverKey
+              ? !activeKeys.includes(s)
               : false
           }
-          key={s.name}
-          name={s.name}
+          key={s}
+          name={s}
           stroke={TOTAL_ENTITY_CHART_COLOR[i] ?? getRandomHexColor()}
           strokeOpacity={
-            isEmpty(activeMouseHoverKey) || s.name === activeMouseHoverKey
+            isEmpty(activeMouseHoverKey) || s === activeMouseHoverKey
               ? DEFAULT_CHART_OPACITY
               : HOVER_CHART_OPACITY
           }
```

Comment:
this is line chart, i dont see changes related to bar chart, is that expected?

Reply:
We want to same logic everywhere for rendering charts and this will resolve date sorting issue as well